lib/mayaUsd/fileio/importData.cpp

```diff
@@ -124,7 +130,7 @@ void ImportData::setStageInitialLoadSet(UsdStage::InitialLoadSet loadSet)
 
 bool ImportData::hasVariantSelections() const
 {
-	return !(fRootVariants.empty() || fPrimVariants.empty());
+	return !(fRootVariants.empty() && fPrimVariants.empty());
```

Comment:
Logic error I noticed.  Don't think it was actually called from anywhere so no existing defect for QA to test.